src/httpGet.tsx

```diff
@@ -0,0 +1,42 @@
+import axios, { AxiosResponse } from 'axios';
+import { URL_BASE, API_KEY, API_KEY_NUMBER } from './Data';
+import type { ResponseData } from './httpGet.types';
+
+export async function httpGet(url: string, params?: Record<string, string>) {
+    // Record -> typ Record<K, T> -> typ OBIEKTU -> gdzie: K === typ kluczy; T === typ wartości 
```

Comment:
This file should be heavily cleaned up - you should not use comments unless they explain stuff that can't be easily understood from the code itself.